packaging/systemd/microshift-containerized.service

```diff
@@ -13,8 +13,10 @@ RequiresMountsFor=%t/containers
 Environment=PODMAN_SYSTEMD_UNIT=%n
 Restart=on-failure
 TimeoutStopSec=70
+ExecStartPre=/bin/mkdir -p /var/lib/microshift # not autogenerated
+ExecStartPre=setsebool container_manage_cgroup true # not autogenerated
 ExecStartPre=/bin/rm -f %t/%n.ctr-id
-ExecStart=/usr/bin/podman run --cidfile=%t/%n.ctr-id --cgroups=no-conmon --rm --replace --sdnotify=container --label io.containers.autoupdate=registry --network=host --privileged -d --name microshift -v /var/run:/var/run -v /sys:/sys:ro -v /var/lib:/var/lib:rw,rshared -v /lib/modules:/lib/modules -v /etc:/etc -v /run/containers:/run/containers -v /var/log:/var/log -e KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig quay.io/microshift/microshift:latest
+ExecStart=/usr/bin/podman run --cidfile=%t/%n.ctr-id --cgroups=no-conmon --rm --replace --sdnotify=container --label io.containers.autoupdate=registry --network=host --privileged -d --name microshift -v /var/run/crio/crio.sock:/var/run/crio/crio.sock:rw,rshared -v /var/lib/microshift:/var/lib/microshift:rw,rshared -v /var/lib/kubelet:/var/lib/kubelet:rw,rshared -v /var/log:/var/log quay.io/microshift/microshift:latest
```

Comment:
These Podman arguments still bind mount the `/var/lib/microshift` data-dir to the host file system instead of using a Podman volume `microshift-data` as described in #433.

Also, it's not clear why we need to bind mount `/var/lib/kubelet` and `/var/log` at all? For the latter, I would have expected the container to simply log to stdout as is common on K8s?